app/models/measure_condition_classification.rb

```diff
@@ -0,0 +1,46 @@
+class MeasureConditionClassification
+  DOCUMENT_CLASS = 'document'.freeze
+  EXEMPTION_CLASS = 'exemption'.freeze
+  THRESHOLD_CLASS = 'threshold'.freeze
+  NEGATIVE_CLASS = 'negative'.freeze
+  UNKNOWN_CLASS = 'unknown'.freeze
+
+  EXCLUDED_DOCUMENT_CODE_CLASS_DOCUMENT_CODES = [
+    '999L', # National Document: CDS universal waiver
+    'C084', # Other certificates: Exemption by virtue of Articles 3 and 4 of regulation 2019/2122 (animals intended for scientific purposes, research and diagnostic samples)
+  ].freeze
+
+  EXEMPTION_CERTIFICATE_TYPE_CODE = 'Y'.freeze # Particular provisions
```

Comment:
we should group this with the other const strings above.

Reply:
I mean its conceptually totally different